### PR TITLE
Legacy docs

### PR DESCRIPTION
**Pull request by diegokolena**
### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

**Comment by munkyshi** [comment]
Updated `hotfix/0.100.x` to be cut from the `0.100.0` tag, rather than an arbitrary point in time from `trunk`. I've gone ahead and cherry picked the changes back to this older point in time, though it might not have updated smoothly, due to code restructuring over the past few weeks.

File: .github/workflows/legacy_docs.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - hotfix/0.100.x
+      - diego/legacy-docs
```
Comment (munkyshi) [inline .github/workflows/legacy_docs.yml]: Should probably remove this before merging.

File: mkdocs.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
-site_name: Testing with Kolena
+site_name: Testing with Kolena (legacy)
```
Comment (munkyshi) [inline mkdocs.yml]: Might want this to say `0.100.0` instead of `legacy`

**Review by munkyshi — APPROVED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (diegokolena) [reply mkdocs.yml]: Updated

File: .github/workflows/legacy_docs.yml (hunk 1 above)
Comment (diegokolena) [reply .github/workflows/legacy_docs.yml]: Updated

**Pull request merged**